examples/00-fluent/modeling_cavitation.py

```diff
@@ -306,7 +303,7 @@
 ###############################################################################
 # Mirror the display around the symmetry plane to show the full model.
 
-solver.tui.display.set.mirror_zones(["symm_2", "symm_1"])
+solver.results.graphics.views.mirror_zones = ["symm_2", "symm_1"]
```

Comment:
:+1: Caveat: we ought to start to use `solver.settings` (also below).

Reply:
Done. Thank you.